notebook/static-src/notebook/js/celltoolbarpresets/example.js

```diff
@@ -8,17 +8,14 @@
 // ```
 // $.getScript('/static/js/celltoolbarpresets/example.js');
 // ```
-define([
-    'jquery',
-    'notebook/js/celltoolbar',
-], function($, celltoolbar) {
     "use strict";
-
+    var celltoolbar = require('notebook/js/celltoolbar');
     var CellToolbar = celltoolbar.CellToolbar;
 
     var example_preset = [];
 
-    var simple_button = function(div, cell) {
+    var simple_button = function(div
```

Comment:
This is just an example so it isn't breaking anything, but this was the only glaring bug I saw.